src/handle/compile.ml

```diff
@@ -88,6 +88,7 @@ let rec compile_with :
       Path.Map.iter compile !(sign.sign_deps);
       loaded := Path.Map.add mp sign !loaded;
       Sign.link sign;
+      Sign.link Unif_rule.sign;
```

Comment:
Ah we just forgot to link the ghost signature! But why do we need to do this? And why each time we compile a file?